.github/workflows/test-query-engine.yml

```diff
@@ -60,6 +60,7 @@ jobs:
             connector: "cockroachdb"
             version: "22.1"
         engine_protocol: [graphql, json]
+        relation_load_strategy: [join, query]
```

Comment:
Should we only do this for providers that support `relationJoins`?

Reply:
Hm, it would reduce ci output clutter at least. I can look into this

Reply:
I know that we don't support `relationJoins` on MySQL versions < 8.0.14 and MariaDB, but neither of those are tested on ci.

I think we can exclude mongodb?
Can we exclude vitess, sqlite, and mssql?
```
- name: "vitess_8_0"
- name: "postgres16"
- name: "mssql_2022"
- name: "sqlite"
- name: "mongodb_4_2"
- name: "cockroach_23_1"
- name: "cockroach_22_2"
- name: "cockroach_22_1_0"
```

Reply:
We can exclude MongoDB, SQLite and SQL Server.

Reply:
> I know that we don't support relationJoins on MySQL versions < 8.0.14 and MariaDB, but neither of those are tested on ci.

They are tested on Buildkite.